src/gc/gcinterface.d

```diff
@@ -0,0 +1,196 @@
+/**
+ * Contains the internal GC interface.
+ *
+ * Copyright: Copyright Digital Mars 2005 - 2016.
+ * License:   $(WEB www.boost.org/LICENSE_1_0.txt, Boost License 1.0).
+ * Authors:   Jeremy DeHaan
+ */
+
+ /*          Copyright Digital Mars 2005 - 2016.
+ * Distributed under the Boost Software License, Version 1.0.
+ *    (See accompanying file LICENSE or copy at
+ *          http://www.boost.org/LICENSE_1_0.txt)
+ */
+module gc.gcinterface;
+import gc.stats;
+
+private
+{
+    static import core.memory;
```

Comment:
Can you do this the other way around - define `BlkInfo` here and import it from `core.memory`? Otherwise there may be cyclic dependency - `core.memory` imports the module and vice versa. Also, is `BlkInfo` common for all possible GCs, or is it specific for the current conservative one?